src/rocky/index.js

```diff
@@ -0,0 +1,43 @@
+var rocky = require('rocky');
+
+var dataLogs = [];
+var memoryPressureHitCount = 0;
+
+// Triggered when memory pressure is an issue
+rocky.on('memorypressure', function(event) {
+  console.log('memory pressure event: ' + event.level);
```

Comment:
Probably best to log this after releasing the pressure, just to be sure that there's enough memory to run this line.